src/main/java/io/strimzi/kafka/metrics/MetricFamilySamplesBuilder.java

```diff
@@ -22,6 +22,12 @@ public class MetricFamilySamplesBuilder {
     private final String help;
     private final List<Collector.MetricFamilySamples.Sample> samples;
 
+    /**
+     * Constructor for MetricFamilySamplesBuilder.
+     *
+     * @param type The type of the metric family.
+     * @param help help string for the metric family.
```

Comment:
`@param help help string for the metric family.` -> `@param help The help string for the metric family.`

Reply:
@mimaison Done